### PR TITLE
Adds feature with servlet.

### PR DESCRIPTION
**Pull request by abundis28**
Probably, in the future will change the <a href="> for a button in the navbar .

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Andrés!</h1>");
```
Comment (timothy-parsons) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Can you update the above comment too to clarify what this servlet is now returning? Also get in the habit of having TODOs written as 'TODO(aabundis)' - it doesn't matter as much here but when the three of you are writing code together as a pod, it'll be useful to be able to see who is supposed to be doing/finishing what at a glance.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -15,9 +15,12 @@ <h3 id="text">This is my portfolio! I will present some of my most important asp
...
       </div>
-      <p id="text">Click here to get a random language that I speak:</p>
-      <button onclick="addRandomLanguage()">Click!</button>
-      <div id="language-container"></div>
+      <div>
+        <p id="text">Click here to get a random language that I speak:</p>
+        <button onclick="addRandomLanguage()">Click!</button>
+        <div id="language-container"></div>
+      </div>
+      <div><p id="text">Click <a href="/data">here</a> to demo here the Java servlet.</p></div>
```
Comment (timothy-parsons) [inline portfolio/src/main/webapp/index.html]: nit: you say 'here' twice in four words. Maybe get rid of the second 'here'.

Comment (timothy-parsons) [inline portfolio/src/main/webapp/index.html]: nit: Consider for consistency's sake moving the <p> element onto the next line, like you do above.

**Review by timothy-parsons — APPROVED** [review]
Some small comments, otherwise looks good!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (abundis28) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Done!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (abundis28) [reply portfolio/src/main/webapp/index.html]: Ack!

Comment (abundis28) [reply portfolio/src/main/webapp/index.html]: Done!

**Pull request merged**